src/expr.rs

```diff
@@ -37,6 +37,10 @@ pub enum SimpleExpr {
     Constant(Value),
 }
 
+pub trait IntoSimpleExpr {
+    fn into_sub_query_expr(self) -> SimpleExpr;
```

Comment:
@alphavector hello! Naming confused me a little bit) 

@tyt2y3 what do you think?

```rust
pub trait IntoSimpleExpr {
   fn into_simple_expr(self) -> SimpleExpr;
}
```

Reply:
Actually, there is only one impl for this trait. May be we can simply make it an inherit method?